simulator/protocols/bk.ml

```diff
@@ -212,15 +219,15 @@ module Make (Parameters : Parameters) = struct
       let open Compare in
       let cmp =
         by int block_height_exn
-        $ by int (fun x ->
-              List.length (Dag.children votes_only x |> List.filter vote_filter))
-        $ by (tuple int int |> neg) leader_hash_exn
+        $ by int (fun x -> List.length (confirming_votes x |> List.filter vote_filter))
```

Comment:
Use children instead of confirming votes. Confirming votes might have been a bad idea, considering all the troubles I had in the tailstorms. Also do this is bkll.

Reply:
It should be fine. `confirming_votes` and `confirmed_votes` assert that the argument is a block.